db/schema.rb

```diff
@@ -510,7 +509,7 @@
     t.string "slug"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["slug"], name: "index_training_slides_on_slug", unique: true
+    t.index ["slug"], name: "index_training_slides_on_slug", unique: true, length: 191
```

Comment:
This one appears to be correct, as ConvertTrainingModulesToUtf8Mb4 contains this constraint.